src/server.rs

```diff
@@ -220,27 +233,37 @@ impl CoAPServer {
     }
 }
 
-fn transmit_handler(tx_recv: RxQueue, tx_only: UdpSocket) {
-    // Note! We should only transmit with this UDP Socket
+fn response_handler(tx_recv: &RxQueue, tx_only: &UdpSocket) {
     // TODO: Add better support for failure detection or logging
```

Comment:
Yeah. I will think about it. And I think the thread_pool isn't needed any more. Because the event pool is a single thread, and the writable event will be emitted one by one.